public/static/js/task.js

```diff
@@ -267,6 +267,25 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
         $scope.taskHost = $window.taskHost;
         $scope.jiraHost = $window.jiraHost;
         $scope.subscriptions = [];
+
+        $scope.validateDuration = function(duration) {
+          if (!Number.isInteger(+duration)) {
```

Comment:
Just do duration = +duration at the top?

Reply:
I'm intentionally not mutating it so that I can return it in error strings